src/test.js

```diff
@@ -1,3 +1,13 @@
+import { extend, runLoggingCallbacks, now, saveGlobal, checkPollution, synchronize, defined, inArray } from "./core";
+import config from "config";
+import sourceFromStacktrace from "sourceFromStacktrace";
+import { extractStacktrace } from "sourceFromStacktrace";
+import Assert from "assert";
```

Comment:
Yeah, those need to be in their own modules, or whatever is necessary to get rid of the circular dependency.